graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -120,12 +120,17 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
    * subgraph.
    *
    * @param nodes The nodes from which to sample neighbors.
-   * @param fanout The number of edges to be sampled for each node. It should be
-   * >= 0 or -1. If -1 is given, it is equivalent to when the fanout is greater
-   * or equal to the number of neighbors and replacement is false, in which case
-   * all the neighbors will be selected. Otherwise, it will pick the minimum
-   * number of neighbors between the fanout value and the total number of
-   * neighbors.
+   * @param fanouts The number of edges to be sampled for each node with or
+   * without considering edge types. The length of it should be 1 or equal
+   * to the number of edge types. A length of 1 means the sampling will be
+   * performed once for each node, regardless of edge types. Otherwise, it
+   * will be performed independently for each edge type. The value of it should
+   * be greater than or equal to 0 or -1. If -1 is given, it is equivalent to
+   * when the fanout is greater or equal to the number of neighbors and
+   * replacement is false, in which case all the neighbors will be selected. In
+   * contrast, a non-negative integer determines the minimum number of neighbors
+   * to select, which is determined by comparing the fanout value with the total
+   * number of neighbors available.
```

Comment:
Let's reorganize this as follow.

```suggestion
   * @param fanouts The number of edges to be sampled for each node with or
   * without considering edge types. The length of it should be 1 or equal
   * to the number of edge types.
   *   - A length of 1 means the sampling will be performed once for each node,
   *     regardless of edge types.
   *   - Otherwise, it will be performed independently for each edge type.
   * The value of it should be...
```

Reply:
Fixed